sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -537,6 +542,24 @@ object Radar {
             override fun onComplete(config: RadarConfig) {
                 locationManager.updateTrackingFromMeta(config?.meta)
                 RadarSettings.setFeatureSettings(context, config?.meta.featureSettings)
+
+                if (config.googlePlayProjectNumber != null) {
+                    // Create an instance of a standard integrity manager
+                    val standardIntegrityManager = IntegrityManagerFactory.createStandard(context)
+                    standardIntegrityManager.prepareIntegrityToken(
```

Comment:
is it safe to call createStandard / prepareIntegrityToken multiple times? Doesn't seem like it should be an issue, but want to make sure if `initialize` is called more than once it doesn't cause trouble. can probably confirm w/ example app

Reply:
Looks like they call out can prepare a token 5 up to times a min https://developer.android.com/google/play/integrity/standard#prepare-integrity

so then follow up question would be around re-warming up. Should we add another prepare token call in the activity resume lifecycle?

Reply:
calling multiple times is ok. 

I've added a warm-up call to `onActivityResumed`